be/src/storage/lake/update_manager.cpp

```diff
@@ -52,6 +52,8 @@ UpdateManager::UpdateManager(LocationProvider* location_provider, MemTracker* me
     _update_state_cache.set_mem_tracker(_update_state_mem_tracker.get());
 
     _index_cache.set_capacity(update_mem_limit);
+    _block_cache = std::make_unique<PersistentIndexBlockCache>(
+            mem_tracker, update_mem_limit * config::lake_pk_index_block_cache_limit);
```

Comment:
Unified the config usage here. Maybe `lake_pk_index_block_cache_limit_percent = 10` is better.

Reply:
And create a new tracker under `_update_mem_tracker`, and set name to it.

Reply:
done